packages/ra-data-graphcool/src/buildVariables.js

```diff
@@ -166,10 +165,11 @@ export default introspectionResults => (
                 queryType
             );
         }
-
         case DELETE:
             return {
                 id: params.id,
             };
+        default:
+            console.log('Unknown fetch verb');
```

Comment:
No, unknown fetch verbs are allowed and can be implemented in custom dataProviders

Reply:
Yes, but it is a good practice to handle the default case right ?

Reply:
Those dataProviders did not handle them at all and returned undefined. How should we handle that ?